lib/wallet/src/Cardano/Wallet/Transaction.hs

```diff
@@ -290,14 +290,17 @@ newtype PreSelection = PreSelection { outputs :: [TxOut] }
 
 data Withdrawal
     = WithdrawalSelf RewardAccount (NonEmpty DerivationIndex) Coin
-    | WithdrawalExternal RewardAccount (NonEmpty DerivationIndex) Coin
+    | WithdrawalExternal
+        RewardAccount
```

Comment:
Now technically redundant when we have the `XPrv`...

Reply:
Or — it depends on how you look at it. From this point of view the `RewardAccount` could technically have a script credential with a single cosigner... I'll avoid commenting anything about this (and avoid potentially big refactoring).